makePoplogCommander.sh

```diff
@@ -241,6 +412,20 @@ poplog --dev [OPTION]...
     normal mode for programming in Poplog. It is not normally necessary to
     supply this option.
 
+poplog --use-build=(nox|xt|xm) [OPTION]...
+    This option selects the build-variant of Poplog. Poplog can be built
+    without X-windows (nox), with the X-Toolkit (xt) or with Motif (xm).
+    This mainly affects the availability of xved and its look and feel.
+    
+    If this option is not specified then the default depends on whether 
+    poplog is being run interactively (--dev) or as a script (--run).
+    In interactive mode, the environment variable $POPLOG_USE_BUILD is checked, 
+    which should have one of the three values nox, xt or xm. Otherwise it 
+    falls back to xm (Motif).
```

Comment:
Is motif still default? I thought you changed that.

Reply:
It is the default now.